### ISSUE TITLE
Unicode in `@string` macros?

### ISSUE THREAD

**Issue by paddymcall** [plk/biber#435]
I have a Biblatex file /tmp/basic.bib like this:

```
@string{P = {Pöblisher}}

@article{key,
  author    = {Authors},
  title     = {Title},
  publisher = P
}
```

When I run it through biber, I end up with this in /tmp/basic_bibertool.bib:

```
@STRING{P = "PÃ¶blisher"}

@ARTICLE{key,
  AUTHOR = {Authors},
  PUBLISHER = P,
  TITLE = {Title},
}
```
Note that the "ö" in the original `@string` has been garbled.  I would have expected biber to not touch the encoding of `@string`  here.

I've tried with Biber 2.18, and also with bin/biber from the dev branch here (https://github.com/plk/biber/blob/d08549cfd5882ce6bbe79eb6d7267ebf2188029e/bin/biber), both with various combinations of `-u` and `-U`.

**Comment by paddymcall** [comment]
Perfect, that fixes it: `perl -I lib/ ./bin/biber --tool /tmp/basic.bib; cat /tmp/basic_bibertool.bib`:

```
@STRING{P = "Pöblisher"}

@ARTICLE{key,
  AUTHOR = {Authors},
  PUBLISHER = P,
  TITLE = {Title},
}
```

Thanks a lot by the way for this wonderful tool!

**Comment by plk** [comment]
Thanks for confirming. I'll update the DEV version binaries later tonight.

**Comment by plk** [comment]
This was indeed a bug, fixed in 2.19 dev branch and also in binaries in the development folder in sourceforce:

https://sourceforge.net/projects/biblatex-biber/files/biblatex-biber/development/binaries/
